### ISSUE TITLE
remove_foreign_key: undefined local variable or method `its'

### ISSUE THREAD

**Issue by moiristo** [SchemaPlus/schema_plus_foreign_keys#9]
Hi,

Your latest release (0.1.5) causes an error when calling 'remove_foreign_key' in migrations. Requiring 'its-it' explicitly in Gemfile fixes it, so I guess it just needs to be required somewhere.

```
undefined local variable or method `its' for #<ActiveRecord::ConnectionAdapters::PostgreSQLAdapter:0x007fc48ce854b0>
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/schema_plus_foreign_keys-0.1.5/lib/schema_plus/foreign_keys/active_record/connection_adapters/abstract_adapter.rb:109:in `get_foreign_key_name'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/schema_plus_foreign_keys-0.1.5/lib/schema_plus/foreign_keys/active_record/connection_adapters/abstract_adapter.rb:123:in `remove_foreign_key_sql'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/schema_plus_foreign_keys-0.1.5/lib/schema_plus/foreign_keys/active_record/connection_adapters/abstract_adapter.rb:70:in `remove_foreign_key'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:665:in `block in method_missing'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:634:in `block in say_with_time'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:634:in `say_with_time'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:654:in `method_missing'
/Users/nedforce/projects/my_project/db/migrate/20160530151530_remove_foreign_key_from_permit_archives.rb:8:in `up'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:611:in `exec_migration'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:592:in `block (2 levels) in migrate'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:591:in `block in migrate'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/connection_adapters/abstract/connection_pool.rb:292:in `with_connection'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:590:in `migrate'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:773:in `migrate'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:1003:in `block in execute_migration_in_transaction'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:1049:in `block in ddl_transaction'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/connection_adapters/abstract/database_statements.rb:213:in `block in transaction'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/connection_adapters/abstract/transaction.rb:184:in `within_new_transaction'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/connection_adapters/abstract/database_statements.rb:213:in `transaction'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/transactions.rb:220:in `transaction'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:1049:in `ddl_transaction'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:1002:in `execute_migration_in_transaction'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:964:in `block in migrate'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:960:in `each'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:960:in `migrate'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:828:in `up'
/Users/nedforce/.rvm/gems/ruby-2.2.2@devcms/gems/activerecord-4.2.6/lib/active_record/migration.rb:806:in `migrate'
```

**Comment by ronen** [comment]
Are you sure you're not using 0.1.4?  The change for 0.1.5 was exactly to include the dependency on `its-it` (see be1f3761841b63145e7b3a2332c1e2c20dea38a8) to avoid this error.

**Comment by ronen** [comment]
d'oh!  my bad.  0.1.5 included the dependency but didn't require the file.  just released v0.1.6 with this fixed

**Comment by moiristo** [comment]
Exactly! Thanks :)
